distribution/feature-packs/server-feature-pack/src/main/resources/licenses/rh-sso/org.keycloak,keycloak-saml-core,2.5.5.Final-redhat-1,Apache Software License 2.0.txt

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```

Comment:
That would be a bug to file against Keycloak separately. I don't know why the license file is HTML. Beyond line-ending normalization, I don't think automatically changing any of the license texts is wise. Doing so could potentially change the meaning of the license.